ClaudiaHernandezGarcia/26Aug/HackerRank/inheritanceII/Arithmetic.java

```diff
@@ -0,0 +1,5 @@
+package inheritanceII;
+
+class Arithmetic {
```

Comment:
hmmmmmm
 ".....A class named Arithmetic with a method named add that takes 2 integers..."

Reply:
Solved